File: clients/typescript/tests/workspaces/go/fizzbuzz.go
package main

func FizzBuzz(n int) string {
	switch {
	case n%15 == 0:
		return "fizzbuzz"
	case n%3 == 0:
		return "fizz"
	case n%5 == 0:
		return "buzz"
	default:
		return fmtInt(n)
	}
}

File: clients/typescript/tests/workspaces/cpp/src/fizzbuzz.cpp
#include "fizzbuzz.hpp"
#include <string>

std::string fizzbuzz(int n) {
  if (n % 15 == 0) return "fizzbuzz";
  if (n % 3 == 0)  return "fizz";
  if (n % 5 == 0)  return "buzz";
  return std::to_string(n);
}
